pages/store/store.wxss

```diff
@@ -10,29 +10,34 @@
 }
 
 .product_list_cell {
-    padding: 5px;
+    display: block;
+    position: relative;
+    justify-content: space-between;
```

Comment:
This attribute only work when `display: flex` or `display: grid` is set on the container